src/translations/libfm-qt_de.ts

```diff
@@ -281,12 +281,12 @@
     <message>
         <location filename="../file-props.ui" line="607"/>
         <source>SetUID</source>
-        <translation type="unfinished">SetUID</translation>
+        <translation>Lege UID fest</translation>
     </message>
     <message>
         <location filename="../file-props.ui" line="614"/>
         <source>SetGID</source>
-        <translation type="unfinished">SetGID</translation>
+        <translation>Lege GID fest</translation>
```

Comment:
Erm ok - wir werden sehen, wie das im Programm aussieht - eventuell sollte man nicht auf Deubel komm raus alles was nicht bei 3 auf dem Baum ist übersetzen - btw. ein Blick in die zugehörigen Sourcen könnte manchmal nicht schaden.

Soll jetzt keine Kritik sein - aber ok, da sollten wir dann eventuell Richtlinien festlegen und im Zweifel auch Sachen auf Suggestion stellen.